### PR TITLE
Fix: Refactored Complete Help Page Code

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR 
1. i have refactored the code for help page 
2. added the variables for colors 
3.  removed the unwanted code 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@app/styles/style.scss [hunk 1]
```diff
@@ -409,6 +413,11 @@ $theme-green-8dc599: #8dc599;
   --border-tertiary-700: #{$theme-violet-1c1d27};
   --border-tertiary-750: #{$theme-violet-1C1D2B};
 
+
+  --text-success-200: #{$theme-green-00A86B};
+
+
```
Comment (itsmdasifraza) [inline src/packages/@app/styles/style.scss:419]: push this to --text block

File: src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte [hunk 2]
```diff
@@ -172,8 +156,8 @@
                 character={post.author?.name?.charAt(0)}
                 width="34px"
                 height="32px"
-                backgroundColor="#1C1D2B"
-                borderColor="#45494D"
+                backgroundColor="var(--bg-tertiary-750)"
+                borderColor="var(--bg-secondary-300)"
               />
               <div style="font-size: 14px; font-weight: 500;">
                 {post.author?.name}
```
Comment (itsmdasifraza) [inline src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte:163]: --border

File: src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte [hunk 3]
```diff
@@ -252,18 +236,19 @@
         />
 
         <div class="d-flex align-items-center gap-2">
-          <!-- <AttachmentIcon
+          <AttachmentIcon
             height={"12px"}
             width={"12px"}
             color={"var(--text-secondary-200)"}
-          /> -->
+          />
```
Comment (itsmdasifraza) [inline src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte:243]: comment this, will uncomment when click behaviour will added

File: src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte [hunk 4]
```diff
@@ -135,7 +117,7 @@
       handleUpvote();
     }}
   >
-    <LeftIcon color={"#FFFFFF"} />
+    <LeftIcon color={"var(--white-color)"} />
     <span class="px-2" style="font-size: 14px;">back</span>
   </div>
```
Comment (itsmdasifraza) [inline src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte:123]: --text-secondary-100

File: src/packages/@app/pages/Help/HelpPage.svelte [hunk 5]
```diff
@@ -21,20 +21,119 @@
...
+      return `${days} day${days > 1 ? "s" : ""} ago`;
+    } else if (hours >= 1) {
+      return `${hours} hour${hours > 1 ? "s" : ""} ago`;
+    } else if (minutes >= 1) {
+      return `${minutes} minute${minutes > 1 ? "s" : ""} ago`;
+    } else {
+      const seconds = Math.floor(diffInSeconds);
+      return `${seconds} second${seconds > 1 ? "s" : ""} ago`;
+    }
+  };
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Help/HelpPage.svelte:130]: push this to utils

File: src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte [hunk 6]
```diff
@@ -530,17 +610,17 @@
...
+    border-color: var(--text-primary-440);
   }
 
   .tag-improved {
-    color: #00a86b; /* I believe you meant this color for "Improved" */
-    border-color: #00a86b;
+    color: var(--text-success-200);
+    border-color: var(--text-success-200);
   }
 </style>
```
Comment (itsmdasifraza) [inline src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte:626]: --border

File: src/packages/@app/styles/style.scss (hunk 1 above)
Comment (Abhishek9503) [reply src/packages/@app/styles/style.scss:419]: Resolved

File: src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte (hunk 3 above)
Comment (Abhishek9503) [reply src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte:243]: Resolved

File: src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte (hunk 4 above)
Comment (Abhishek9503) [reply src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte:123]: Resolved

File: src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte (hunk 6 above)
Comment (Abhishek9503) [reply src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte:626]: Resolved

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**